terasoluna-tourreservation-web/src/main/resources/IPAfont00303/Readme_IPAfont00303.txt

```diff
@@ -0,0 +1,40 @@
+IPAフォント（IPAフォント4書体パック）
```

Comment:
- [ ] Really is Readme_IPAfont00303.txt required?

Reply:
22ade0bbfd4710d8e8c13e3e15ae4b0f0b8c7370

I removed in above commit .